### PR TITLE
Jin 2020 Markers

### PR DESCRIPTION
**Pull request by rnmitchell**
PR for adding MH markers from Jin 2020.

### CODE REVIEW

File: dbbuild/prep-dbs.sh [hunk 1]
```diff
@@ -7,12 +7,12 @@ download_1kgp()
...
     mkdir -p ${rootdir}/1000Genomes/
     for i in {1..22}; do
-        url="${rooturl}/ALL.chr${i}.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz"
+        url="${rooturl}/ALL.chr${i}.phase3_shapeit2_mvncall_integrated_v5b.20130502.genotypes.vcf.gz"
         filename="${rootdir}/1000Genomes/chr${i}.vcf.gz"
         wget -O ${filename} ${url}
         wget -O ${filename}.tbi ${url}.tbi
     done
-    url="${rooturl}/ALL.chrX.phase3_shapeit2_mvncall_integrated_v1b.20130502.genotypes.vcf.gz"
+    url="${rooturl}/ALL.chrX.phase3_shapeit2_mvncall_integrated_v1c.20130502.genotypes.vcf.gz"
```
Comment (rnmitchell) [inline dbbuild/prep-dbs.sh:15]: When I first tried to download the 1000G VCF files, it erred out. Looks like they've updated this?? It works with the change.

**Comment by rnmitchell** [comment]
Also, when trying to install MicroHapDB, I had to pin a version (7.0.1) of snakemake in order to get it to install properly with conda otherwise it just got hung up forever. I suggest pinning a version.

**Comment by rnmitchell** [comment]
There's also no ```pip install -e .``` directions in installation but it is required.... so should add that too.

File: dbbuild/prep-dbs.sh (hunk 1 above)
Comment (standage) [reply dbbuild/prep-dbs.sh:15]: I haven't tested this in a while, so I'll give it a look during the review.

**Review by standage — APPROVED** [review]
Data and tests look good, and I was able to reproduce the build on my end. Thanks!

**Pull request merged**